x-pack/legacy/plugins/code/public/components/integrations/integrations.scss

```diff
@@ -41,3 +41,6 @@
   margin-bottom: 1rem;
   width: 300px;
 }
+
+
+@import "./code_flyout.scss";
```

Comment:
If you think these styles are temporary with the integrations page, I would put them in the integrations.scss itself. If they're intrinsic to the Flyout, I would export this file from the main style manifest.